server.js

```diff
@@ -6,8 +6,8 @@ const app = express();
 const mongoose = require('mongoose');
 require('dotenv').config();
 
-mongoose.connect(`mongodb://${process.env.MONGO_USER}:${process.env.MONGO_PASS}@localhost:27017/LiveDB`);
-
+const connectionString = `mongodb://${process.env.MONGOUSER}:${process.env.MONGOPASS}@${process.env.MONGO_ENDPOINT}`;
```

Comment:
Just a heads up. I changed this to use environment variables instead of the hardcoded endpoints. No changes are needed.